src/AVFoundation/Enums.cs

```diff
@@ -943,7 +970,8 @@ public enum AVCaptureLensStabilizationStatus : long
 		Unavailable = 4
 	}
 
-	[NoWatch, NoTV, iOS (11,0)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
+	[NoWatch, NoTV, Mac (10,15), iOS (11,0)]
```

Comment:
this is another case we a mistake was made, the enum is only used for iOS API (not macOS API)

It should have a `[NoMac]` attribute (under `#if !XAMCORE_4_0`) but adding a `[Mac (10,15)]` is actually making our old mistake worse (since it increase the metadata size without providing any benefit).

When doing bindings for a new Xcode it's easier to split "new bindings" and "fix bindings" in separate PR. It's much more time consuming to _fix_ things since you can't trust headers (full of lies) and you need to look at API usage and git's history to double check everything.